README.md

```diff
@@ -24,6 +24,12 @@ Replace `D:\host\path\where\to\output\singularity\image` with a path on the host
 
 `docker2singularity` uses the Docker daemon located on the host system. It will access the Docker image cache from the host system avoiding having to redownload images that are already present locally.
 
+### Contributed Examples
+The following are a list of brief examples and tutorials generated by the Singularity community for using **docker2singularity**. If you have an example of your own, please [let us know](https://www.github.com/singularityware/docker2singularity/issues)!
+
+ - [docker2singularity-demo](https://github.com/stevekm/docker2singularity-demo): an example of using docker2singularity on MacOS using Vagrant, complete with notes and a nice Makefile
```

Comment:
What about: "an example of using docker2singularity on MacOS and using Vagrant to test the output Singularity image, complete with notes and a nice Makefile"